sycl/source/kernel_bundle.cpp

```diff
@@ -429,7 +466,7 @@ make_kernel_bundle_from_source(const context &SyclContext,
     IncludePairs.push_back({p.first.data(), p.second.data()});
 
   backend BE = SyclContext.get_backend();
-  if (!is_source_kernel_bundle_supported(BE, Language))
+  if (!is_source_kernel_bundle_supported(BE, Language, SyclContext))
```

Comment:
There is no need to pass `BE` anymore. Callee should get it from the context itself.